### ISSUE TITLE
Futura Theme doesn't look as expected (primary colors are too bright)

### ISSUE THREAD

**Issue by kirillzh** [standardnotes/forum#472]
Expected:
![expected](https://user-images.githubusercontent.com/5660531/53541593-6940b380-3acf-11e9-8d11-861f4f1c9444.jpg)

Actual:
![screen shot 2019-02-27 at 8 36 23 pm](https://user-images.githubusercontent.com/5660531/53541618-82496480-3acf-11e9-8e5c-5bf781b85c4d.png)

**Comment by mobitar** [comment]
The image above is outdated. Themes have underwent major changes since then. But the scrollbar in the second screenshot should be themed, strange that it's not. What OS and version of the app is this? Can you try uninstalling the theme and re-installing and see if that fixes the scrollbars?

**Comment by kirillzh** [comment]
Scrollbar issue seem to happen with all editors and all themes. I'm using Standard Notes Version 3.0.6 (3.0.6) on macOS Mojave 10.14.3 (18D109).
![screen shot 2019-02-28 at 10 54 27 am](https://user-images.githubusercontent.com/5660531/53590721-481ba980-3b47-11e9-8e33-ba9acb5b96bc.png)
![screen shot 2019-02-28 at 10 51 01 am](https://user-images.githubusercontent.com/5660531/53590722-481ba980-3b47-11e9-8148-c0da9bd97c7d.png)

**Comment by kirillzh** [comment]
Also had weird issue when resizing the window (just made it smaller to force scrollbars and made the window large again expecting scrollbars to disappear).

![screen shot 2019-02-28 at 10 55 58 am](https://user-images.githubusercontent.com/5660531/53590896-9cbf2480-3b47-11e9-9b15-fd5d755f7403.png)

**Comment by mobitar** [comment]
Ah, you probably have this set to "Always" in your System Preferences/General: 

<img width="452" alt="screen shot 2019-03-01 at 11 57 48 am" src="https://user-images.githubusercontent.com/3277844/53656576-4cb09280-3c19-11e9-95fe-e5534e085c89.png">

Gives scrollbars a background for some reason. Whereas if it were set to automatically, it doesn't add background, and blends in more nicely.

**Comment by mobitar** [comment]
Regarding resizing issue, it might be the editor margins. Hover over right side of editor and you should see a vertical bar appear that you can drag to resize.

**Comment by kirillzh** [comment]
You are right, I had my scroll bars always showing. Setting it to "When scrolling" seemed to fix the issue, it's themed nicely now and when scrolling, also can't reproduce resizing issue anymore. Not sure why editor gets resized when scroll bars are always showing.

**Comment by mobitar** [comment]
I don't believe the editor resizing issue is related to scrollbars, but most likely related to "margin resizers" within the SN app. There is a very rare bug that I can't reproduce where expanding your window will not updated editor margins, so it will look smaller than the window. If this happens, you can just hover over the right side of the furthest margin of the editor, and drag towards the edge of the window.
